model/src/main/kotlin/OrtIssue.kt

```diff
@@ -48,7 +48,12 @@ data class OrtIssue(
         /**
          * The issue's message.
          */
-        val message: String
+        val message: String,
+
+        /**
+         * The issue's severity.
+         */
+        val severity: Severity = Severity.ERROR
 ) {
     override fun toString() = "${if (timestamp == Instant.EPOCH) "n/a" else timestamp.toString()}: $source - $message"
```

Comment:
The string represenation should also include the severity. Or do you plan to do this in a separate PR because of the refactoring effort involved in changing this?

Reply:
I didn't plan for it at all as I simply forgot about it 😆 About doing it in a separate PR sounds like a good idea to me. Should we do it like that?

Reply:
Yes, I would prefer that.